help/destinations/catalog/advertising/magnite-batch.md

```diff
@@ -0,0 +1,190 @@
+{style="table-layout:auto"}
+
+## Export Type and Frequency {#export-type-frequency}
+
+| Item | Type | Notes | 
+|-----------------------------|----------|----------|
+| Export type | Audience export | You are exporting all members of an audience with the identifiers (name, phone number, or others) used in the Magnite Streaming: Batch destination. |
+| Export frequency | Batch | Batch destinations export files to downstream platforms in increments of three, six, eight, twelve, or twenty-four hours. Read more about batch [file-based destinations](../../../destinations/destination-types#file-based). |
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+Once your destination usage has been approved and Magnite Streaming has shared your credentials, please follow the below steps to authenticate, map, and share data.
+
+### Authenticate to destination {#authenticate}
+
+Locate the Magnite Streaming: Batch destination in the Adobe Experience catalog. Click the additional options button (\...) and then configure the destination connection/instance.
+
+If you already have an existing account, you can locate it by changing the Account type option to "Existing account". Otherwise, you will create an account below:
+
+To create a new account, and authenticate it to the destination for the first time, fill in the required "S3 access key" and "S3 secret key" fields (provided to you via your account manager), and select **[!UICONTROL Connect to destination]**
+
+![destination configuration auth fields unfilled](../../assets/catalog/advertising/magnite/destination-batch-config-auth-unfilled.png)
+
+**Note:** Magnite Streaming's security policy requires a regular rotation of S3 keys. You should expect to update your account in the future with new S3 access and S3 secret keys. You only need to update the account itself - destinations using that account will automatically use the updated keys. Failure to upload the new keys will result in the data failing to send to this destination.
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination connection/instance in the
+  future.
+*  **[!UICONTROL Description]**: A description that will help you identify this
+  destination connection/instance in the future.
+  *  **[!UICONTROL Name of your source partner]**: The name you would like to go by as a source in Magnite Streaming's platform
+
+![destination configuration auth fields filled](../../assets/catalog/advertising/magnite/destination-batch-config-auth-filled.png)
+
+**Note:** If you plan to send multiple ID types (GAID, IDFA, etc.) using the Batch destination, a new destination connection/instance is required for each. Please contact your Magnite Account representative for more information.
```

Comment:
Please use the Note formatting as shown here https://experienceleague.adobe.com/en/docs/contributor/contributor-guide/writing-essentials/markdown#note-blocks